### ISSUE TITLE
File Centipede incapable of adding existent torrent

### ISSUE THREAD

**Issue by cyberhardt** [filecxx/FileCentipede#571]
The program can't add a torrent that already exists - it says "file exists" and then proceeds to download it as if it wasn't there. I mentioned before of a lack of "recheck" and I believe this stems from the same issue. Could it be the protocol? If this uses aria2 for torrenting, it might be - maybe aria2 lacks this function. If that's the case, changing to bittorrent protocol could solve it

**Comment by filecxx** [comment]
I'll add an option: Use recheck instead of popping up an alert.
(FileCentipede uses [libtorrent](https://github.com/arvidn/libtorrent) for torrenting)

**Comment by cyberhardt** [comment]
> I'll add an option: Use recheck instead of popping up an alert. (FileCentipede uses [libtorrent](https://github.com/arvidn/libtorrent) for torrenting)

yeah, recheck is useful not only to check if the torrent is alright but also to check a file that was already downloaded. I'd like to move my torrents to file centipede but I'll have to wait then
